dynlink.raml

```diff
@@ -0,0 +1,216 @@
+#%RAML 0.8
+title: Dynamic link resource definition
+version: v0.1.1-20180620
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+
+
+schemas:
+  - baseline: !include schemas/oic.core-schema.json
+  - batch: !include schemas/oic.batch-retrieve-schema.json
+  - linklist: !include schemas/oic.oic-linklist-schema.json
+  - linkupdate: !include schemas/oic.if.linkupdate-schema.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: [oic.if.ll", "oic.if.b", "oic.if.baseline", "oic.if.linkupdate"]
+
+
+/DynlinkResURI?if=oic.if.baseline:
+    description: |
+      Baseline representation of a dynlink contains only its common properties
+    displayName: dynlink-baseline
+
+    get:
+        description: |
+          Common properties of the dynlink resource type
+        responses:
+          200:
+            body:
+              application/json:
+                schema: baseline
+                example: |
+                {
+                  "rt": ["oic.r.linkupdate"],
```

Comment:
I've never seen this Resource Type defined.  oic.if.linkupdate is an Interface name but not an "rt", right?